src/main/java/fi/metatavu/dcfb/server/locations/LocationConsts.java

```diff
@@ -0,0 +1,12 @@
+package fi.metatavu.dcfb.server.locations;
+
+public class LocationConsts {
+
+  public static final String COORDINATE_REFERENCE_SYSTEM = "epsg4326";
+  public static final String[] SUPPORTED_COORDINATE_REFERENCE_SYSTEMS = {"epsg4326"}; 
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Use already-defined constant 'COORDINATE_REFERENCE_SYSTEM' instead of duplicating its value here. ([squid:S1192](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=11&issues=AWQt_bhzYwZU9vvrs3K2&open=AWQt_bhzYwZU9vvrs3K2)

[ISSUE_KEY]: AWQt_bhzYwZU9vvrs3K2